app/schemas/models/interactives/common/interactive_types.schema.js

```diff
@@ -15,27 +16,29 @@ const interactiveDraggableOrderingSchema = {
         additionalProperties: false,
         properties: {
           text: { type: 'string' },
-          elementId: schema.objectId({ hidden: true })
+          elementId: schema.stringID({ readOnly: true })
         }
       }
     },
-    solution: solutionSchema.elementOrderingSolutionSchema
+    solution: solutionSchema.elementOrderingSolutionSchema,
+    i18n: { type: 'object', format: 'i18n', props: ['labels', 'elements'], description: 'Help translate this interactive.' }
```

Comment:
For my understanding. How do you store the i18n data for a property like elements (which has its own text property).
Does `utils.i18n` work with this format?

Reply:
Yeah, it lets you save the i18n for all the properties in the elements object (i.e. `text` in this case since `elementId` is readOnly).
I also tested that `utils.i18n` does work with this format.